airbyte-cdk/python/airbyte_cdk/base_python/cdk/streams/auth/oauth.py

```diff
@@ -59,9 +59,9 @@ def get_access_token(self):
     def token_has_expired(self) -> bool:
         return pendulum.now() > self._token_expiry_date
 
-    def get_refresh_request_body(self) -> Mapping[str, any]:
+    def get_refresh_request_body(self) -> Mapping[str, Union[str, Any]]:
```

Comment:
isn't `Union[str, Any]` covered by `any`? why is the type changing here? it's a bit throwing off to do `Union[str, Any]` when `Any` could suffice imo

Reply:
You are right. `Any` works.